config/config.ini

```diff
@@ -32,6 +32,12 @@ watch_kube_controller: True
 # Namespace to look for the kube controller pods
 kube_controller_namespace: openshift-kube-controller
 
+# Set to True for the cerberus to monitor kube-scheduler
+watch_kube_scheduler: True
```

Comment:
Kube scheduler needs to be added to monitored components section of the README.md ( https://github.com/openshift-scale/cerberus#kubernetesopenshift-components-supported ).